bin/concat_tables.py

```diff
@@ -0,0 +1,236 @@
+    },
+    "kaiju": {
+        "file": str,
+        "percent": float,
+        "reads_count": int,
+        "taxon_id": str,
+        "taxon_name": str,
+        "RPM": float,
+        "cami_taxid": str,
+        "cami_rank": str,
+        "cami_taxpath": str,
+        "cami_taxpathsn": str,
+        "cami_percentage": float,
+    },
+    "kraken2": {
+        "percentage_fragments_covered": float,
+        "num_fragments_covered": int,
+        "reads_count": int,
+        "rank_code": str,
+        "taxid": str,
+        "sci_name": str,
+        "RPM": float,
+        "cami_taxid": str,
+        "cami_rank": str,
+        "cami_taxpath": str,
+        "cami_taxpathsn": str,
+        "cami_percentage": float,
+    },
+}
+
+
+def parse_args(argv=None):
+    """Define and immediately parse command line arguments."""
+    parser = argparse.ArgumentParser(
+        description="Concatenate kaiju-cami, kraken2-cami and centrifuge-cami tsv tables into one file",
+        epilog="Example: python concat_tables.py -j kaiju.tsv -k kraken2.tsv -c centrifuge.tsv -o SRR12875570_pe-SRR12875570.tsv",
+    )
+    parser.add_argument(
+        "-j",
+        "--kaiju-file",
+        metavar="kaiju-file",
+        type=Path,
+        help="Tsv file with kaiju table combined with cami table",
+    )
+    parser.add_argument(
+        "-k",
+        "--kraken2-file",
+        metavar="kraken2-file",
+        type=Path,
+        help="Tsv file with kraken2 table combined with cami table",
+    )
+    parser.add_argument(
+        "-c",
+        "--centrifuge-file",
+        metavar="centrifuge-file",
+        type=Path,
+        help="Tsv file with kraken2 table combined with cami table",
+    )
+    parser.add_argument(
+        "-o",
+        "--output-file-name",
+        metavar="Path",
+        type=Path,
+        default=Path("concatenated.tsv"),
+        help="The output tsv file name for the concatenated table",
+    )
+    parser.add_argument(
+        "-l",
+        "--log-level",
+        help="The desired log level (default WARNING).",
+        choices=("CRITICAL", "ERROR", "WARNING", "INFO", "DEBUG"),
+        default="WARNING",
+    )
+    return parser.parse_args(argv)
+
+
+def read_classifier(classifier: tuple[Path, str]) -> tuple[pd.DataFrame, str]:
+    """Read classifier tsv into a DataFrame
+
+    Args:
+        classifier (tuple[Path, str]): Tuple with Path to classifier tsv and name of the classifier
+
+    Returns:
+        pd.DataFrame: DataFrame with classifier data
+    """
+    classifier_name: str = classifier[1]
+    data_types: dict = DATA_TYPES.get(classifier_name)
+    logger.info("Reading %s tsv file into a DataFrame", classifier_name)
+    return (
+        pd.read_table(classifier[0], index_col=False, dtype=data_types),
+        classifier_name,
+    )
+
+
+def process_df(classifier: tuple[pd.DataFrame, str]) -> pd.DataFrame:
+    """Rename DataFrame columns, add classifier name column and drop 'file' column from kaiju df
```

Comment:
Where does the `file` column come from?

Reply:
The file column is the first one in the kaiju output tsv files:
https://github.com/genomic-medicine-sweden/gmsmetapost/blob/dev/assets/tsvs/SRR12875558_se-SRR12875558-kaiju.tsv